### PR TITLE
feat: Add translations for dutch and german

### PR DESCRIPTION
**Pull request by lucacasonato**
Also updated some grammar and punctuation for the english localizations

### CODE REVIEW

File: package.nls.json [hunk 1]
```diff
@@ -5,16 +5,16 @@
...
-  "diagnostic.fix.add_ext_name": "Add `.ts` extension.",
-  "diagnostic.fix.use_HTTPS_module": "Use HTTPS module.",
-  "diagnostic.fix.create_module": "Create module.",
-  "diagnostic.fix.fetch_module": "Fetch module.",
-  "diagnostic.fix.lock_std_version": "Lock Deno standard library version.",
-  "diagnostic.report.use_HTTPS_module": "For security, we recommend using the HTTPS module.",
-  "diagnostic.report.lock_std_version": "We recommend you use a locked Deno standard library version.",
-  "diagnostic.report.can_not_found_module": "Cannot found module `{0}`."
+  "err.not_install_deno": "Could not find `deno` in your $PATH. Please install `deno`, then restart the extension.",
+  "diagnostic.fix.add_ext": "Add `.ts` extension",
```
Comment (axetroy) [inline package.nls.json]: `diagnostic.fix.add_ext ` and `diagnostic.fix.use_https_module` and `diagnostic.report.use_https_module` have been removed.

Can you update this part

**Review by axetroy — CHANGES_REQUESTED** [review]

File: package.nls.json [hunk 2]
```diff
@@ -5,16 +5,13 @@
   "deno.command.disable": "Disable Deno",
   "deno.command.restart_server": "Restart Deno Language Server",
   "deno.config.enabled": "Controls whether Deno is enabled or not.\n\n**Not recommended in global configuration**",
-  "deno.config.dtsFilepaths": "The file paths of the TypeScript declaration file(.d.ts).\n\nIt can be a relative which path relative to the project directory or an absolute path.\n\n**Not recommended in global configuration**",
-  "deno.config.import_map": "The file paths of Import Map.\n\nIt can be a relative which path relative to the project directory or an absolute path.\n\n**Not recommended in global configuration**",
+  "deno.config.dtsFilepaths": "The file path of the Deno TypeScript declaration file (.d.ts).\n\nIt can be a path that is relative to the project root, or an absolute path.\n\n**Not recommended in global configuration unless you are using an absolute path.**",
+  "deno.config.import_map": "The file path of a import map.\n\nIt can be a path that is relative to the project root, or an absolute path.\n\n**Not recommended in global configuration**",
```
Comment (David-Else) [inline package.nls.json]: It should be **an** import map

File: package.de-de.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  "deno.command.disable": "Deaktiviere Deno",
+  "deno.command.restart_server": "Starte den Deno Language Server neu",
+  "deno.config.enabled": "Steuert, ob Deno aktiviert ist oder nicht.\n\n**In der globalen Konfiguration nicht empfohlen**",
+  "deno.config.dtsFilepaths": "Der Dateipfad der Deno TypeScript-Deklarationsdatei (.d.ts).\n\nDies kann ein Pfad sein, der relativ zum Projektpfad ist, oder ein absoluter Pfad.\n\n**Wird in der globalen Konfiguration nicht empfohlen, ausser wenn Sie einen absoluten Pfad verwenden.**",
+  "deno.config.import_map": "Der Dateipfad einer import map.\n\nDies kann ein Pfad sein, der relativ zum Projektpfad ist, oder ein absoluter Pfad.\n\n**In der globalen Konfiguration nicht empfohlen**",
+  "deno.initializing": "Deno Language Server wird initialisiert ...",
+  "err.not_install_deno": "Konnte `deno` in Ihrem $PATH nicht finden. Bitte installieren Sie `deno` und starten Sie die Erweiterung neu.",
+  "diagnostic.fix.create_module": "Erstellen das Modul",
+  "diagnostic.fix.fetch_module": "Download das Modul",
+  "diagnostic.fix.lock_std_version": "Sperren Sie das Modul auf die aktuelle Deno-Version",
```
Comment (axetroy) [inline package.de-de.json]: Can you remove this as it is no longer used

- `diagnostic.fix.lock_std_version `
- `diagnostic.report.lock_std_version `

**Review by axetroy — CHANGES_REQUESTED** [review]

File: package.de-de.json (hunk 3 above)
Comment (lucacasonato) [reply package.de-de.json]: I must say, I really liked these... oh well.

**Review by axetroy — APPROVED** [review]
Nice work.

LGTM & thx.

**Pull request merged**